mapsims/data/example_config.toml

```diff
@@ -21,7 +21,10 @@ num = 0
     LA_number_LF = 1
     LA_number_MF = 4
     LA_number_UHF = 2
-    SA_years_LF = 1
+    SA_years = 4.99657768652
```

Comment:
Can we add a comment in the toml that this number is due to the leap year correction?